docs/tutorials/pre_executed/map_partitions.ipynb

```diff
@@ -54,24 +54,18 @@
    "cell_type": "code",
```

Comment:
<p>here it is Pandas DataFrame, before just dataframe. Lets make it consistent within a notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/792/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Sounds good. What do you think of using <code>pd.DataFrame</code> (in code font) throughout? </p>